modules/nodes-pods-adjust-resources-in-place-about.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-adjust-resources-in-place.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nodes-pods-adjust-resources-in-place-about_{context}"]
+= About in-place pod resizing
+
+In-place pod resizing allows you to change the CPU and memory resources for containers within a running pod without application disruption. The standard methods for changing pod CPU and memory resources cause the pod to be re-created, potentially causing disruption. In-place pod resizing allows you to scale pod resources up or down without suffering the downtime or state loss associated with a pod restart.
+
+When using in-place pod resizing to change CPU or memory resources, you can control whether a pod is restarted by configuring a resize policy in the pod specification. The following example resize policy requires a pod restart upon changing the memory resources, but prevents a restart for CPU resource changes.
+
+.Example resource policy
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: resize-demo
+spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+  - name: pause
+# ...
+    resizePolicy: <1>
+    - resourceName: cpu
+      restartPolicy: NotRequired
+    - resourceName: memory
+      restartPolicy: RestartContainer
+----
+<1> Specifies a resize policy. 
+
+[NOTE]
+====
+Memory limits cannot be decreased unless the resize policy for `memory` is `RestartContainer`.
+====
+
+You cannot add or modify a resize policy to an existing pod, but you can add or edit the policy in the pod's owner object, such as a deployment, if the pod has an owner object. 
+
+Using in-place pod resizing requires that you use the `--subresource resize` flag when editing a pod in the {oc-first}, as shown in the following examples:
+
+.Example commands
+[source,terminal]
+----
+$ oc edit pod <pod_name>  --subresource resize
+----
+
+[source,terminal]
+----
+$ apply -f <file_name>.yaml --subresource resize
+----
+
+[source,terminal]
```

Comment:
I'm not sure if we need outputs for these. I mention this because I feel like I heard this was part of the 2.0 CQA work or JTBD. I could've hallucinated this too. I couldn't find anything firm on it so this isn't a blocker, IMO.